events/bingo/serverScripts/bingoWheel/bingoWheelServer.js

```diff
@@ -12,20 +12,26 @@
 
 (function() {
     var _this;
+    
+    // START EXTERNAL INCLUDES
+    var request = Script.require(Script.resolvePath('../../modules/request.js')).request;
+    // END EXTERNAL INCLUDES
 
     var LIGHT_BLINK_INTERVAL_MS = 500;
     var REMOVE_CARDS_SCRIPT = Script.resolvePath("../../entityScripts/cardRemover/bingoCardRemover.js");
     var CARD_SPAWNER_SCRIPT = Script.resolvePath("../../entityScripts/cardSpawner/bingoCardSpawner.js");
     var WAIT_FOR_ENTITIES_TO_LOAD_MS = 1000;
-    var WAIT_WHILE_CARDS_ARE_DELETED_MS = 3000;
-    var SPREADSHEET_URL = Script.require(Script.resolvePath('../../secrets/bingoSheetURL.json')).sheetURL;
+    var REQUEST_URL = Script.require(Script.resolvePath('../../config/config.json?' + Date.now())).requestURL;
+    var DB_TABLE_PREFIX = Script.require(Script.resolvePath('../../config/config.json?' + Date.now())).dbTablePrefix;
     var BINGO_WALL = "{df198d93-a9b7-4619-9128-97a53fea2451}";
```

Comment:
if a user imports our domain backup, there's a tiny chance it could be different (but it's really small). right now, hardcoded IDs are the best way of identifying other entities.